connector/oidc/oidc.go

```diff
@@ -425,6 +431,28 @@ func (c *oidcConnector) createIdentity(ctx context.Context, identity connector.I
 				}
 			}
 		}
+		c.logger.Debugf("Groups::: %q \n", groups)
+
+		// Validate that the user is part of allowedGroups
+		var accessAllowed = false
+		if len(c.allowedGroups) > 0 {
+			for _,group := range c.allowedGroups {
+				groupPresent := slices.Contains(groups, group)
+				if groupPresent {
+					accessAllowed = true
+					c.logger.Debugf("Found a match - Group::: %q \n", group)
```

Comment:
We prefer structured logging over custom message formats

Reply:
Sure, could you point me to an example?

Reply:
Handled in 14f867af2fa25cd4a440c7779431223ab55796af